components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -168,6 +178,34 @@ private void init(String projectName, String configPath, String deploymentConfig
             }
             String deploymentConfigPath = CmdUtils.getDeploymentConfigLocation(projectName);
             ContainerConfig containerConfig = TOMLConfigParser.parse(deploymentConfigPath, ContainerConfig.class);
+            if (isDocker) {
+                PrintStream outStream = System.out;
+                String dockerName = CmdUtils.promptForTextInput(outStream, "Enter docker image name: ").trim();
```

Comment:
If --docker option is provided it always prompt as it seems. Would n't it be better if we provide command-line args for these as well?

Reply:
Thank you for the suggestion. Fixed in https://github.com/wso2/product-microgateway/pull/865/commits/623b75a3da642e0f0142eace77e98ecded0d1fbf